### DISCUSSION TITLE
Is it possible to show which versions are not affected in vulnerability scan results

### DISCUSSION THREAD

**Discussion by nancheal** [DependencyTrack/dependency-track#2541] [Q&A]
Sometimes, it is necessary to know which version of the component to upgrade to fix the vulnerability, so can the version number of the fix be displayed in the vulnerability scanning result?

**Comment by stevespringett** [comment]
OSS Index provides this data for authenticated users, but not over their API, and therefore not with Dependency-Track. Use the OSS Index attribution link, authenticate to OSS Index, and you'll see the available versions and their vulnerability count. This will help you determine what version to upgrade to.
